index.d.ts

```diff
@@ -275,6 +259,37 @@ type PageTypeType = {
 	isSensitive: boolean;
 };
 
+// Data types for the API request bodies from clients. The 'CAPI' prefix
+// convention doesn't really make sense anymore (it is not the CAPI model,
+// though closely related) but is not worth changing now.
+
+interface CAPILinkType {
+    url: string;
+	title: string;
+    longTitle: string;
+    iconName: string;
+	children?: CAPILinkType[];
+	mobileOnly?: boolean;
+	pillar?: CAPIPillar;
+    more?: boolean;
+    classList?: string[];
+}
+
+interface CAPINavType {
+    currentUrl: string;
+    pillars: CAPILinkType[];
+	otherLinks: CAPILinkType[];
+	brandExtensions: CAPILinkType[];
+    currentNavLink?: CAPILinkType;
+    currentParent?: CAPILinkType;
+    currentPillar?: CAPILinkType;
+	subNavSections?: {
+        parent?: CAPILinkType;
+        links: CAPILinkType[];
+    };
+	readerRevenueLinks: ReaderRevenuePositions;
```

Comment:
some unusual spacing here

Reply:
ooh yeah! Thanks. Will fix. Strange my auto-formatting didn't rescue me.

Reply:
It turned out to be hidden tabs that appeared the same width as 4 spaces locally 😱

Reply:
I've had the same issue with auto-formatting recently. `yarn prettier:fix` should help